src/DataTransferObjects/LogLine.php

```diff
@@ -0,0 +1,140 @@
+<?php
+declare(strict_types=1);
+
+namespace LoyaltyCorp\Auditing\DataTransferObjects;
+
+use DateTime;
+use LoyaltyCorp\Auditing\DataTransferObject;
+
+class LogLine extends DataTransferObject
+{
+    /**
+     * Client ip
+     *
+     * @var string
+     */
+    private $clientIp;
+
+    /**
+     * Occurred timestamp
+     *
+     * @var \DateTime
+     */
+    private $occurredAt;
+
+    /**
+     * Request data
+     *
+     * @var array|mixed[]
+     */
+    private $requestData;
+
+    /**
+     * Response data
+     *
+     * @var array|mixed[]
+     */
+    private $responseData;
+
+    /**
+     * Status
```

Comment:
This field is for indicating if we've synced this line to Elastic search or not. It would be good to add a comment to this effect.

Reply:
added